sensing/autoware_imu_corrector/src/gyro_bias_estimator.cpp

```diff
@@ -464,10 +464,10 @@ void GyroBiasEstimator::update_rate_ekf(
     // Scale on x , y axis is not estimated, but set to 1.0 for consistency
     vector_scale.vector.x = 1.0;
     vector_scale.vector.y = 1.0;
-    vector_scale.vector.z = ekf_angle_.filtered_scale_angle_;
+    vector_scale.vector.z = 1 / ekf_angle_.filtered_scale_angle_;
```

Comment:
Please add a check to prevent divide by zero on all of the added division operations.